src/nrfx/hal/nrf_clock.h

```diff
@@ -310,6 +376,19 @@ static inline nrf_clock_lfclk_t nrf_clock_lf_src_get(NRF_CLOCK_Type const * p_re
     return (nrf_clock_lfclk_t)(p_reg->LFCLKSRC);
 }
 
+static inline nrf_clock_lfclk_t nrf_clock_lf_srccopy_get(NRF_CLOCK_Type const * p_reg)
```

Comment:
Please add these 2 after nrf_clock_is_running() so they are ordered like in the original HAL file (so it is much easier to find differences)